### ISSUE TITLE
[bug]: Brian is missing `draftForHighBust` text

### ISSUE THREAD

**Issue by jdhoek** [freesewing/freesewing#3083]
### What seems to be the problem? 🤔

[Brian](
https://freesewing.org/designs/brian/) has an option called `draftForHighBust`. This option is currently missing text in both title and description.

![Screenshot from 2022-11-19 12-55-37](https://user-images.githubusercontent.com/683699/202849582-2dc6863a-6fd2-4fe3-8841-e3f40a9f979e.png)

![Screenshot from 2022-11-19 12-55-53](https://user-images.githubusercontent.com/683699/202849598-61d222c8-1346-4162-8519-4ba755b723ca.png)

### Design / Plugin / Package 🧐

designs/brian

### Are you a FreeSewing patron? 😃

No, I am not 😞

### Additional context

Text/translations for these seem to be missing:

* `options.brian.draftForHighBust.title`
* `options.brian.draftForHighBust.description`

**Comment by BenJamesBen** [comment]
This proposed fix would copy the wording from Teagan to `packages/i18n/src/locales/en/options/brian.yml`:
```diff
@@ -26,6 +26,10 @@ cuffEase:
   title: Cuff ease
   description: The amount of ease at your wrist.
 
+draftForHighBust:
+  title: Draft for high bust
+  description: Draft the pattern for the high bust measurement (if available) rather than the (full) chest. This will result in a more fitted garment for people with breasts.
+
 frontArmholeDeeper:
   title: Front armhole extra cutout
   description: How much do you want the front armhole to be cut out deeper than the back.
```

**Comment by bobgeorgethe3rd** [comment]
closed as of #3789
